examples/chart/teleport-kube-agent/README.md

```diff
@@ -13,14 +13,7 @@ To use it, you will need:
 - a reachable proxy endpoint (`$PROXY_ENDPOINT` e.g. `teleport.example.com:3080` or `teleport.example.com:443`)
 - a reachable reverse tunnel port on the proxy (e.g. `teleport.example.com:3024`). The address is automatically
   retrieved from the Teleport proxy configuration.
-- either a static or dynamic join token for the Teleport Cluster
-  - a [static join token](https://goteleport.com/docs/setup/admin/adding-nodes/#adding-nodes-to-the-cluster)
-    for this Teleport cluster (`$JOIN_TOKEN`) is used by default.
-  - optionally a [dynamic join token](https://goteleport.com/docs/setup/admin/adding-nodes/#short-lived-dynamic-tokens) can
-    be used on Kubernetes clusters that support persistent volumes. Set `storage.enabled=true` and
-    `storage.storageClassName=<storage class configured in kubernetes>` in the helm configuration to use persistent
-    volumes.
-
+- a join token for the Teleport Cluster. For this Teleport cluster (`$JOIN_TOKEN`) is used by default. See the [join token guide](https://goteleport.com/docs/enroll-resources/agents/join-services-to-your-cluster/join-token/) for information on creating tokens.
```

Comment:
Could you link to the token reference instead as it explains how to create any kind of token and links to all the how-tos, including this one: https://goteleport.com/docs/reference/join-methods/

Reply:
Sure @hugoShaka , please see update.